src/Resources/contao/templates/backend/be_pagination.html5

```diff
@@ -1,5 +1,6 @@
 
-<div class="pagination <?= $this->class ?>">
+<nav class="pagination <?= $this->class ?>" role="navigation" aria-label="<?= $GLOBALS['TL_LANG']['MSC']['paginationAriaLabel'] ?>">
+  <a href="<?= $this->request ?>#skipNavigation" class="invisible"><?= \StringUtil::specialchars($GLOBALS['TL_LANG']['MSC']['skipNavigation']); ?></a>
```

Comment:
At first sorry for my late reply.

To your statement above: I think thats the same like her: https://github.com/contao/core-bundle/pull/868#issuecomment-307322372

We are talking about the backend here, aren't we?